source-gallery/src/main/java/com/google/android/apps/muzei/gallery/GalleryImportPhotosDialogFragment.java

```diff
@@ -82,7 +65,46 @@ public void onDetach() {
         mListener = null;
     }
 
+    @Override
+    public void onStart() {
+        super.onStart();
+        List<ActivityInfo> newGetContentActivities = mListener.getGetContentActivityInfoList();
+        if (newGetContentActivities == null || newGetContentActivities.isEmpty()) {
+            // User un-install all of the apps that can handle the GET_CONTENT action
+            this.dismiss();
+            return;
+        }
+
+        if (newGetContentActivities.size() == mGetContentActivities.size()) {
```

Comment:
@ianhanniballake Let me know if this is OK if not I can always update the adapter